src/components/Table/ExpandedDataRow.js

```diff
@@ -129,6 +134,10 @@ class ExpandedDataRow extends PureComponent {
             </div>
           ))}
         </div>
+        {/* Custom footer component */}
+        <div className={theme.expandedRow.subDataContainer}>
+          {CustomFooterComponent}
```

Comment:
Couldn’t this just be `children` and let this be an implementation detail? There’s no real need to have a specialized component for this is there?